docs/book/src/tasks/experimental-features/cluster-classes.md

```diff
@@ -0,0 +1,497 @@
+# Cluster Class
+
+The ClusterClass feature introduces a new way to create clusters which reduces boilerplate required and allows flexible and powerful customization of CAPI-created clusters.
+ClusterClass is implemented on top of pre-existing CAPI interfaces and will 
+
+This tutorial covers using Cluster API and Cluster Class to create a Kubernetes cluster and how to perform a one-touch Kubernetes update procedure using ClusterClass features.
+
+<aside class="note warning">
+
+<h1>Note</h1>
+
+This guide covers creating a ClusterClass cluster with the CAPD provider.
+</aside>
+
+## Installation
+
+### Common Prerequisites
+This guide is based on having the same pre-requisites as the CAPI [quick start guide].
+That is:
+- Install and setup [kubectl]  and [clusterctl]
+- Install [Kind] and [Docker]
+
+ClusterClass is currently behind a feature gate that needs to be enabled. To do so run:
+
+```bash
+export  EXP_CLUSTER_RESOURCE_SET=true
+export CLUSTER_TOPOLOGY=true
+```
+This ensures that Cluster Topology and Cluster Resource Set are enabled when we initialize our CAPI Providers.
+*Note*: Cluster Resource Set is not required for ClusterClass but is used in this tutorial to set up networking.
```

Comment:
```suggestion

*Note*: Cluster Resource Set is not required for ClusterClass but is used in this tutorial to set up networking.
```
Looks like this needs an additional new line, otherwise it will be rendered without line break in the same line as the sentence above.

Fyi, not sure if you're aware. You can view the book locally (with auto-refresh) via:
```bash
cd  ./docs/books
make serve
# http://localhost:3000/tasks/experimental-features/cluster-classes.html
# I think the link only works after adding the file to SUMMARY.md)
```

Reply:
I didn't really think putting the Note on its own line was needed here, but I'll update.

Reply:
Ah okay, just thought that was intended as it was already on a new line in the markdown